kalite/main/models.py

```diff
@@ -279,7 +279,10 @@ def add_log_to_summary(cls, user_log, device=None):
             start_datetime__lte=user_log.end_datetime,
             end_datetime__gte=user_log.end_datetime,
         )
-        assert log_summary.count() <= 1, "There should never be multiple summaries in the same time period/device/user/type combo"
+
+        if log_summary.count() > 1:
+            for log in log_summary[1:]:
+                log.soft_delete()
```

Comment:
@jamalex ,
I will add a Todo.

The possible case of the issue could be : (Not very sure)
We have disabled the automatic logout and we have our summary duration as of 1 month.
So, during the last date of month say 30 or 31st Dec, if the account is not logged out then we could have two log summary one for Dec and one for Jan and both would be the same.